application-research/estuary
# Restify /contents API

Addresses #583

## Verification
1. Add CAR, add to bucket (new name for collections) and dir too
![image](https://user-images.githubusercontent.com/8129788/204874891-b09b4ab9-774c-4fb2-9a30-64de5a9527f8.png)
1.2. List that bucket on that dir to see the file
![image](https://user-images.githubusercontent.com/8129788/204875189-965d2098-8479-4b0a-8ad5-c2b1cd1675a7.png)

2. Add CID, add to bucket and dir too
![image](https://user-images.githubusercontent.com/8129788/204876037-2aebaa7f-dc5f-4d7b-8a81-abc03d8a12ae.png)
2.2. List that bucket on that dir to see the pinned file
![image](https://user-images.githubusercontent.com/8129788/204876116-84e4e3f7-73eb-41d9-90e1-6cb02b09e4c8.png)

3. Getting the status of one of the contents
![image](https://user-images.githubusercontent.com/8129788/204883667-c3212941-0c05-4657-ae36-ce13d275ebeb.png)

4. Get content by CID
![image](https://user-images.githubusercontent.com/8129788/204884137-a81abcb0-cf89-451b-ab35-da82d5c8af99.png)
